### ISSUE TITLE
FR: Allow deleting/hiding seen notifications

### ISSUE THREAD

**Issue by maniac103** [openhab/openhab-cloud#178]
As per the subject, it seems a little counterintuitive that it's impossible to ever remove old notifications. It would be great to have a UI button and API to remove them, ideally with varying granularity (single/all/up to certain date).

**Comment by broyuken** [comment]
I agree with this, I set up a rule to send a few test notifications to make sure it works, but now I can't remove them. As such my notifications are now full of "junk" notifications that I can't get rid of.

**Comment by digitaldan** [comment]
I bulked closed all issues older then a year since i just rewrote the cloud service and don't have time to go through all the old issues as i'm the only one who maintains this project and it not my main focus by any means.  Just FYI, we only store 30 days worth of notifications on the server. You can open this up again if you plan on contributing something, i don't plan on working on this given the 30 day deletion policy.
